domain/src/memory.js

```diff
@@ -86,6 +88,17 @@ function selectNotDeleted(dict) {
 }
 
 /**
+ * Полностью обновляет предмет(кроме id)
+ * @returns {Promise.<Number, Error>} Количество найденных объектов
+ */
+async function updateItemFully(db, coll, id, item) {
+    if (await database.get_by_id(db, coll, id)) {
+        await database.add_by_id(db, coll, id, item);
```

Comment:
What you are doing here is much trickier than it seems by the first glance. Essentially it is a transaction https://en.wikipedia.org/wiki/Database_transaction. Don't put too much thought into it for now, but writing the db-related code like this will lead to a veeeeeeery nasty bugs in the future. I've created issue about that https://github.com/emotz/kladovka/issues/76.